include/kernels/dslash_wilson.cuh

```diff
@@ -144,7 +156,7 @@ namespace quda
     static constexpr const char *filename() { return KERNEL_FILE; } // this file name - used for run-time compilation
 
     // out(x) = M*in = (-D + m) * in(x-mu)
-    __device__ __host__ inline void operator()(int idx, int s, int parity)
```

Comment:
The removal of `s` here makes it "impossible" to apply this to 4D slices of a 5D spinor (such as the two-flavour). I've done this: https://github.com/qcdcode/quda/commit/045992381836aaef5f87340c1d4fcc07e2f75556 to be able to call `WilsonApply` on a 5D field, but I'm not sure if this aligns with your strategy here.

Reply:
If you’re using it, just restore it.  I removed for compiler warning.

Reply:
Thanks. I'm not sure about the other change that I make in that commit in the Wilson constructor:

```
    Wilson(Arg &arg, const ColorSpinorField &out, const ColorSpinorField &in) : Dslash(arg, out, in)
    {
      if(in.Ndim() == 5) {
        TunableVectorYZ::resizeVector(in.X(4), arg.nParity);
      }
    }
```

Reply:
Not sure I follow?  Looks fine to me

Reply:
> Not sure I follow? Looks fine to me

Sorry, I should have been more verbose in my question. I was just wondering if this is the "correct" way to deal with the `Ndim() == 5` case for general `in.X(4)` (if `WilsonApply` is ever called on a 5D spinor from a DWF run, for example). I followed the template of what is done for the two-flavour operators, but I wasn't certain if this generalises acceptably for `in.X(4) > 2` and I don't want to cause headaches for others down the line.

Reply:
Sorry, I missed answering this: yes this generalizes fine.